DTS-STN/canadian-dental-care-plan
# Moving route helper server modules to `.server` directory

### Description
As per title.

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/1cdd37ba-83bd-4007-b7eb-0523d16ad295)

### Checklist

### Test Instructions
Run the application and verify that nothing breaks for apply, renew, and protected renew flows as well as hCaptcha.
